server.js

```diff
@@ -148,30 +176,34 @@ function generateChart(request, callback) {
             .append('path')
             .attr('class', 'line')
             .style('stroke', (d, i) => `#${lineColors[i] || '2ba8de'}`)
+            .style('stroke-width', '2')
```

Comment:
this we can do in the css above probably. the stroke is just down here since it is dynamic